core/src/main/java/io/questdb/cairo/mv/MatViewState.java

```diff
@@ -141,6 +158,10 @@ public void close() {
         cursorFactory = Misc.free(cursorFactory);
     }
 
+    public long getLastPeriodHi() {
```

Comment:
Please document `Period Hi` and how it's used for different types of refreshes.

Reply:
Done in 1972434fefd3f811845b436079fe96cab966e985